app.js

```diff
@@ -33,50 +34,65 @@ readableStream.on('end', function() {
 
 // Switch Model
 // Expects an object:{
-  // id:"sw" + number,
-  // state: "on" or "off",
-  // name: any name you want to display. Defaults to "switch"
+// id:"sw" + number,
+// state: "on" or "off",
+// name: any name you want to display. Defaults to "switch"
```

Comment:
kas te mainījās? Atliksi atpakaļ kā bija?

Reply:
Nē, šis lai paliek